src/main/default/classes/LookupSearchResult.cls

```diff
@@ -17,6 +17,10 @@ public class LookupSearchResult implements Comparable {
         this.subtitle = subtitle;
     }
 
+    public LookupSearchResult(String id, String title, String icon) {
```

Comment:
quick question: the sobject type is missing here, what is it used for, why can we skip it?

Reply:
I don't have all the fields in my use-case, that is why I have created another constructor